lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2114,6 +2114,9 @@ Instruction *SPIRVToLLVM::transEnqueueKernelBI(SPIRVInstruction *BI,
     SPIRVConstant *NE = static_cast<SPIRVConstant *>(NumEvents);
     HasEvents = NE->getZExtIntValue() != 0;
   }
+  SPIRVValue *EventRet = Ops[5];
+  if (EventRet->getOpCode() == OpPtrCastToGeneric)
```

Comment:
Why `OpPtrCastToGeneric` ?

Reply:
I'd prefer: `if (EventRet->getOpCode() != OpConstantNull)`. It's more readable.